src/Control/Monad/Bayes/Inference/RMSMC.hs

```diff
@@ -3,14 +3,14 @@
 
 -- |
 -- Module      : Control.Monad.Bayes.Inference.RMSMC
--- Description : Resample-Move Sequential Monte Carlo (RM-SMC)
+-- Description : Resample-Move SequentialT Monte Carlo (RM-SMC)
 -- Copyright   : (c) Adam Scibior, 2015-2020
 -- License     : MIT
 -- Maintainer  : leonhard.markert@tweag.io
 -- Stability   : experimental
 -- Portability : GHC
 --
--- Resample-move Sequential Monte Carlo (RM-SMC) sampling.
+-- Resample-move SequentialT Monte Carlo (RM-SMC) sampling.
 --
```

Comment:
These changes should be removed

Reply:
could you describe the proposed impact of this work?
as a research fellow without a project I have infinitely many hours to invest, but I value resourcefulness, efficiency and sustainability. 
Figuring out one line of code that would implement significant change and satisfy the requirements was fun and interesting task.
Cleaning up the mess is simpler with "close request" button.

Do you guys need a technical writer at Tweag? Because during an interview 6 months ago I came across a technical requirement that looked like the author hated every second of the process.

Reply:
> could you describe the proposed impact of this work?

You mean this PR here? It's about improving maintainability of the library and decreasing surprise for Haskell developer discovering this library, to enable them to program with less friction.

> Do you guys need a technical writer at Tweag?

About that I have no idea, but I guess Github is the wrong place to inquire that. I'd rather recommend heading to the careers page on the Tweag website.